src/services/BackendService/BackendService.ts

```diff
@@ -56,7 +58,19 @@ export class BackendService implements BackendInterface {
     const region = 'CA';
     const exposureConfigurationUrl = `${this.retrieveUrl}/exposure-configuration/${region}.json`;
     captureMessage('getExposureConfiguration', {exposureConfigurationUrl});
-    return (await fetch(exposureConfigurationUrl, FETCH_HEADERS)).json();
+    const timeoutSecs = Number(Config.DOWNLOAD_TIMEOUT_SECONDS);
+
+    const abortController = new AbortController();
+    const opts = Object.assign(FETCH_HEADERS, {signal: abortController.signal});
+
+    if (timeoutSecs > 0) {
```

Comment:
`getExposureConfiguration` is the only `fetch` that we use in js layer. We can move it to native layer via CovidShield Module for consistency.

Reply:
I would leave as is.

Reply:
moved to native layer now